src/plugins/intel_gpu/src/graph/impls/ocl_v2/lora_opt.cl

```diff
@@ -276,24 +278,3 @@ KERNEL(first_token_b)(OPTIONAL_SHAPE_INFO_ARG
     ADD_AND_STORE_CODE
 }
 #endif
-
-#ifdef FUSED_OPS_KERNEL
-KERNEL(fused_ops)(OPTIONAL_SHAPE_INFO_ARG
-                  __global OUTPUT_TYPE* output
-#if HAS_FUSED_OPS_DECLS
-                , FUSED_OPS_DECLS
-#endif
-                    )
-{
-    const uint b = get_global_id(0);
-    const uint f = get_global_id(1);
-    const uint y = get_global_id(2) / OUTPUT_SIZE_X;
-    const uint x = get_global_id(2) % OUTPUT_SIZE_X;
-    const uint output_idx = OUTPUT_GET_INDEX(b, f, y, x);
-
-#if HAS_FUSED_OPS
-    FUSED_OPS;
-    output[output_idx] = FUSED_OPS_RESULT;
-#endif
-}
-#endif
```

Comment:
May I ask how are you going to do the post ops fusing?

Reply:
Just like before, fused kernel is no longer duplicated in the opt kernel, but is used from the reference kernel